2020-epc-dev/demo-embed-arcgis/src/apps/DispatchApp.js

```diff
@@ -74,7 +76,9 @@ function UserDispatchApp(){
           <Container>
             <h2>Dispatch app!</h2>
             <p>Assign maintenance requests to workers</p>
-            <Button half onClick={onLogin}>Log in</Button>
+            <Button half onClick={onLogin}>Log in</Button><br/><br/>
+            {UserName}<br/>
+            {Password}
```

Comment:
Just hard-code the values here rather than importing as a variable
```js
<Button half onClick={onLogin}>Log in</Button><br/>
{epcdemo}</br>
{epcpw}
```